src/js/components/Analytics.jsx

```diff
@@ -43,9 +45,17 @@ class Analytics extends Component {
   }
 
   componentWillMount () {
-    const {dispatch} = this.props
+    const {dispatch, user} = this.props
+    const event = {
+      group: 'pageView',
+      attributes: {
+        page: 'Analytics',
+        user: user.data.username
+      }
+    }
 
     dispatch(fetchSources())
+    dispatch(sendMetrics([event]))
```

Comment:
We may be able to move this into the onEnter hook of the Router in index.jsx.

https://github.com/reactjs/react-router/blob/master/examples/auth-flow/app.js#L133